discordb.py

```diff
@@ -23,7 +23,7 @@
     sys.exit(1)
 
 # Discord message size limit.
-DISCORD_MESSAGE_SIZE_LIMIT = 4096
+DISCORD_MESSAGE_SIZE_LIMIT = 3000
```

Comment:
This is actually what the original value was (2000). No idea why I changed it, I suspect it was to match the Slack backend I was shamelessly copying.

https://discordapp.com/developers/docs/resources/channel#create-message

Reply:
I tested the 3000 value when I committed and it appeared to work, but a second test returned HTTP 400 from the Discord API.  I think it would be best to set it back to 2000 to avoid people getting intermittent errors.